_config/logging.yml

```diff
@@ -38,6 +38,7 @@ SilverStripe\Core\Injector\Injector:
       - "error"
     properties:
       Formatter: %$FriendlyErrorFormatter
```

Comment:
I could just use `SilverStripe\Logging\DebugViewFriendlyErrorFormatter` as the injection point